.github/workflows/release-charts.yaml

```diff
@@ -0,0 +1,106 @@
+name: Release Helm Charts
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      HELM_SIGNING_KEY: ${{ secrets.TEST_SIGN_HELM }}
+      HELM_SIGNING_KEY_ID: ${{ secrets.HELM_SIGNING_KEY_ID }}
+      GITHUB_REPOSITORY: ${GITHUB_REPOSITORY}
+      PYTHONPATH: /opt/hostedtoolcache/Python/3.9.14/x64/lib/python3.9/site-packages
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.9.4
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9.14'
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip gitpython ruamel_yaml
+
+      - name: Generate Release Notes for GitHub Releases
+        run: |
+          cd src/main/scripts
+          products=(bamboo bamboo-agent bitbucket confluence crowd jira)
+          export REPO_PATH="${GITHUB_WORKSPACE}"
+          export CHARTS_LOCATION="src/main/charts"
+          for product in "${products[@]}"; do
```

Comment:
Upload another md file to each chart tgz so that chart releaser can upload release notes to GitHub releases page.